Detectors/Base/include/DetectorsBase/Utils.h

```diff
@@ -42,11 +44,11 @@ namespace AliceO2 {
         c = cos(ang);
```

Comment:
@ktf Hi Giulio, this is from my old code: not sure to which extent the sincos/sincosf are portable: these are GNU extensions. I added this sincosf intending eventually to check for the presence of library function, but have not done this. Agree for double / float cast: there should be call to sinf, cosf respectively.

Cheers,
 Ruben